github/examples_test.go

```diff
@@ -73,7 +73,7 @@ func ExampleRepositoriesService_CreateFile() {
 	ctx := context.Background()
 	fileContent := []byte("This is the content of my file\n\nand the 2nd line of it")
 
-	// Note: the files need to be absent from the repository as you are not
+	// Note: the file need to be absent from the repository as you are not
```

Comment:
How about `the file needs` ?
Also, how about adding a period after `reference here.`?